travis/test.sh

```diff
@@ -21,6 +21,8 @@ flutter test --flutter-repo
 
 if [ $TRAVIS_PULL_REQUEST = "false" ]; then
   if [ $TRAVIS_BRANCH = "master" ]; then
+    pub global activate dartdoc 0.8.4
+    export PATH="$PATH":"~/.pub-cache/bin"
```

Comment:
Instead, of manipulating the path, please just run the `~/.pub-cache/bin/dartdoc` directly.

Also, you shouldn't ever use `~` in the PATH environment variable.  Not all consumers of PATH understand how to expand `~`.  The fact that the pub tool tells you to use this string is a bad recommendation.